File: server/Models/Content.cs
using System;
using System.Collections.Generic;
using System.Runtime.CompilerServices;

public class Content
{
    //Consturoctor yapısı

    // public Content(){} 
    // public Content(List<Section> sections,List<Navigation> navigations)
    // {
    //     this.Sections = sections;
    //     this.Navigation = navigations;
    // }
    // var c = new Content(

    //         new List<Section>{new Section{Header = "Dogukan Koc"}},
    //         new List<Navigation>{new Navigation{Logo = "Dogukan Koc"}}
        
    //     );
    
    //Bu bölüm ise aslında sitemin içerdiği kısımların bir kopyası diyebiliriz. Yazısal olarak da sitemi içeriyor aslında. Çünkü sitemde
    //Öncelikle yandaki menüde bir logo bulunuyor.

    //public List<string> Headers { get; set; } // Projelerim Ozgecmis CV

    public List<Section> Sections { get; set; }
    public List<Navigation> Navigation { get; set; }
}

File: web/src/model/content.ts
export default interface Content {
  Navigation: Navigation[];
  Sections: Section[];
}

export interface Navigation {
    Logo?: string;
    Href?: string;
    Text?: string;
    Target?: string;
    Icon?: string;
  }
  
export interface Section {
  Header?: string;
  Text?: string;
  Name?: string;
  Title?: string;
  Description?: string;
  Href?: string;
}

  